src/Features/Core/Portable/Extensions/IExtensionDocumentMessageHandlerWrapper.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.CodeAnalysis.Extensions;
+
+/// <summary>
+/// Wrapper for an <c>IExtensionDocumentMessageHandler</c> as returned by <see cref="IExtensionMessageHandlerFactory"/>. 
```

Comment:
`<see` instead of `<c>`?

Reply:
I can't use `see` because the Features project doesn't have a reference to ExternalAccess where these interface are defined